Blog + BlogPost (#211)
enjoythevue/website: blog/why-theres-no-new-episode-this-week.md

---
title: Why There's No New Episode This Week
publish: 2020-06-01T10:19:30
slug: why-theres-no-new-episode-this-week
---

> Due to current events happening in the United States regarding George Floyd and the protests to bring about changes to systemic racial injustice and police brutality, we will not be releasing a new episode this week and will be donating our sponsorship profits to [Campaign Zero](https://www.joincampaignzero.org/) from last week's episode.

Reviewer: In support of the recent protests to bring about changes to systemic racial injustice and police brutality [instigated by the murder of George Floyd? Not sure if we want to call out one death when there were so many?], we will will be donating our sponsorship profits from last week's episode to [Campaign Zero](https://www.joincampaignzero.org/) from last week's episode.

Author: ```suggestion
In support of the protests to bring about changes to systemic racial injustice and police brutality in the United States, we will not be releasing a new episode this week and will be donating our sponsorship profits to [Campaign Zero](https://www.joincampaignzero.org/) from last week's episode.
```

What do you think of this @tessaSac?

Reviewer: > To support the US protests against systemic racism and police brutality, we will be donating last week's sponsorship money to Campaign Zero, and refrain from releasing an episode this week in order to focus on the global conversation on these grave injustices.

Might be more in line with the changes to our tweet?

If you would also like to lend your support, here are a couple ways you can do so.

1. Donate

- [Campaign Zero](https://www.joincampaignzero.org/)
- [NAACP Legal Defense Fund](https://www.naacpldf.org/)
- [Minnesota Freedom Fund](https://minnesotafreedomfund.org/)
- [Black Visions Collective](https://secure.everyaction.com/4omQDAR0oUiUagTu0EG-Ig2)
- [Reclaim the Block](https://secure.everyaction.com/zae4prEeKESHBy0MKXTIcQ2)

> 2. Read about it

Author: ```suggestion
2. Read about the topic
```

> 2. Read about the topic

Author: ```suggestion
2. Read
```

- [How To Be Antiracist, Ibram X Kendi](https://bookshop.org/books/how-to-be-an-antiracist/9780525509288)
- [So You Want To Talk About Race, Ijeoma Oluo](https://bookshop.org/books/so-you-want-to-talk-about-race/9781580058827)

> Thanks for your understanding and we hope you all stay safe amidst these trying times.

Reviewer: Thank you for understanding; we hope you all are all safe and well? Sounds a bit like a run on without at least a comma or something 🤔

Author: ```suggestion
Thanks for your understanding. We hope you all stay safe amidst these trying times.
```

What do you think @tessaSac?
